crates/xtask/src/bump.rs

```diff
@@ -31,10 +31,10 @@ impl BumpCommand {
             .windows(2)
             .all(|w| w[0].version == w[1].version)
         {
-            anyhow!(
+            return Err(anyhow!(
```

Comment:
Can we just use `bail!` here? I haven't been too worried about `xtask` since it is essentially repo-internal scripts, but thanks for fixing this!